src/components/scroll-view/scroll-view.js

```diff
@@ -181,36 +160,16 @@ export default class ScrollView extends Component {
       fullWidth,
       queryListName,
       prevNextButtons,
-      offset: page,
-      isMainPageSearch,
       length,
     } = this.props;
 
-    const {
-      offset,
-      visibleItems,
-    } = this.state;
-
-    let listHeight = length || items.length;
-
-    if (isMainPageSearch) {
-      listHeight = items.length <= (PAGE_SIZE * page)
-        ? items.length % PAGE_SIZE
-        : PAGE_SIZE;
-    }
-
-    // list height should be at least enough for the offset
-    if (listHeight === 0) {
-      listHeight = offset + visibleItems;
-    }
-
-    listHeight *= itemHeight;
+    const listHeight = (length || items.length) * itemHeight;
```

Comment:
What if `length` will be more then 100?  Isn't list will have empty space?

Reply:
This prop wasn't actually passed to the component from anywhere so I removed it

Reply:
Ok, but if `items.length` will be more then 100?